crates/rolldown/tests/rolldown/function/external/double-namespace-reexport/main.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable */
```

Comment:
cc @Boshen. The `.oxlintignore` already ignored `crates/**` path, but it will failed at `git commit -a -m`.I run `npm run lint-code` is ok. I'm not sure it is has some different...

Reply:
I met this too https://discord.com/channels/1138376133969203262/1199281741165248522/1287715251135053895